torchvision/csrc/io/decoder/defs.h

```diff
@@ -165,7 +165,7 @@ struct MediaFormat {
 struct DecoderParameters {
   // local file, remote file, http url, rtmp stream uri, etc. anything that
   // ffmpeg can recognize
-  std::string uri;
+  std::string uri{std::string()};
```

Comment:
Why this? 
(don't know C++ well, so I'm just curious)

Reply:
just want to initialise  the uri value as empty string